.github/workflows/ecosystem-tools.yaml

```diff
@@ -61,10 +61,13 @@ jobs:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ github.token }}
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
       - name: Install localstack
         run: |
           pip install --upgrade pyopenssl
-          pip install localstack==2.3.1 awscli-local[ver1] # install LocalStack cli and awslocal
+          pip install localstack==3.3.0 awscli-local[ver1] # install LocalStack cli and awslocal
           docker pull ghcr.io/project-zot/ci-images/localstack:2.3.1  # Make sure to pull a working version of the image
```

Comment:
```suggestion
          docker pull ghcr.io/project-zot/ci-images/localstack:3.3.0  # Make sure to pull a working version of the image
```
?

Reply:
What is the concern here?

Reply:
You are updating the client but not the server?
There is a version mismatch between them.

Reply:
Fixed.